includes/Classifai/Helpers.php

```diff
@@ -302,3 +302,70 @@ function get_feature_taxonomy( $feature ) {
 	 */
 	return apply_filters( 'classifai_taxonomy_for_feature', $taxonomy, $feature );
 }
+
+/**
+ * Provides the max filesize for the Computer Vision service.
+ *
+ * @since 1.4.0
+ *
+ * @return int
+ */
+function computer_vision_max_filesize() {
```

Comment:
Moved these functions out of ComputerVision class because they're now called from multiple places and don't use ComputerVision class state.